<launch>
    <executable
        cmd="ros2 bag play --clock 1000 $(find-pkg-share gyak3)/bag"
        output="screen"
        name="rosbag_play"
        shell="true"
    />

    <node pkg="tf2_ros" exec="static_transform_publisher" name="map_to_robot2_odom" output="screen"
        args="0.0 0.0 0.0 0.0 0.0 0.0 map odom"/>

    <node pkg="my_parking_robot" exec="waypoint_sender" name="waypoint_sender_node" output="screen">
        <param name="text_file_name" value="/home/andrisgerber/ros2_workspace/src/my_parking_robot/param/path.txt"/>
    </node> 

    <node pkg="my_parking_robot" exec="waypoint_follower" name="waypoint_follower_node" output="screen">
        <param name="publish_rate" value="10.0"/>
    </node>

    <node pkg="rviz2" exec="rviz2" name="rviz2" args="-d $(find-pkg-share my_parking_robot)/rviz/my_parking_robot.rviz"/>

</launch>